x/payment/keeper/keeper.go

```diff
@@ -8,19 +8,36 @@ import (
 
 	"github.com/celestiaorg/celestia-app/x/payment/types"
 	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 )
 
 const payForDataGasDescriptor = "pay for data"
 
 // Keeper handles all the state changes for the celestia-app module.
 type Keeper struct {
-	cdc codec.BinaryCodec
+	cdc        codec.BinaryCodec
+	storeKey   storetypes.StoreKey
+	memKey     storetypes.StoreKey
+	paramstore paramtypes.Subspace
```

Comment:
[uber nit][optional] proposal to camelCase this variable globally because the variable is composed of two words. The proposal conforms to the naming convention used for the other fields in this struct (i.e. `storeKey` and `memKey`)

```suggestion
	paramStore paramtypes.Subspace
```